lib/api/filter.rb

```diff
@@ -6,7 +6,15 @@ class Filter
       ">=" => {:default => ">="},
       "<"  => {:default => "<", :datetime => "BEFORE"},
       ">"  => {:default => ">", :datetime => "AFTER"},
-      "="  => {:default => "=", :datetime => "IS", :regex => "REGULAR EXPRESSION MATCHES", :null => "IS NULL"}
+      "="  => {:default => "=", :datetime => "IS", :regex => "REGULAR EXPRESSION MATCHES", :null => "IS NULL"},
+
+      # string-only matching, use quotes
+      "==" => {:default => "="},
+      "!==" => {:default => "!="},
```

Comment:
ugh that's unfortunate that we need `==` and `!==`

Reply:
maybe `===` and `!==` would be better for symmetry?

I was also thinking of leaving the operators as they are and forcing the value by using the right kind of quoting character (requiring quotes for strings and slashes for regexes), but that would break existing code.

Reply:
w.r.t. == vs. ===, not sure, maybe we can do something similar to js w/ type checking.

I guess if we are doing strict type checking requiring value to be quoted, i.e. attr==="value" or attr==='value' and attr column type must be string, the === and !== might be better. 

This way we could introduce == and != in the future which would give us attr=="5" and attr==5  both being true for integer type fields.